From: SeongJae Park <sj@kernel.org>
Date: Sun, 20 Jul 2025 09:19:55 -0700
Subject: [PATCH] selftests/damon/sysfs.py: reduce watermarks check time to
 500ms

In usual test setup, the watermarks parameter makes DAMON be disabled by
the watermarks.  Since the watermarks check time interval is 5 seconds,
next commit can be done at least after the 5 seconds.  This makes the
test to take unnecessarily long time.  Make it faster, by reducing the
watermarks check time interval to 500 milliseconds.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 tools/testing/selftests/damon/sysfs.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/damon/sysfs.py b/tools/testing/selftests/damon/sysfs.py
index d8b152eec174..b2cb178dda15 100755
--- a/tools/testing/selftests/damon/sysfs.py
+++ b/tools/testing/selftests/damon/sysfs.py
@@ -218,7 +218,7 @@ def main():
                     weight_nr_accesses_permil=200,
                     weight_age_permil=1000),
                 watermarks=_damon_sysfs.DamosWatermarks(
-                    metric = 'free_mem_rate', interval = 5000000, # 5 secodns
+                    metric = 'free_mem_rate', interval = 500000, # 500 ms
                     high = 500, mid = 400, low = 50),
                 target_nid=1,
                 apply_interval_us=1000000,
-- 
2.39.5

